commons/ava_services/gecko_service_availability_checker_core.go

```diff
@@ -22,13 +22,13 @@ func (g GeckoServiceAvailabilityCheckerCore) IsServiceUp(toCheck services.Servic
 	// HACK HACK HACK we need to wait for bootstrapping to finish, and there is not API for this yet (in development)
 	// TODO TODO TODO once bootstrapping checker is available, use that instead of just waiting
 	if healthInfo.Healthy {
-		time.Sleep(15 * time.Second)
+	 	time.Sleep(15 * time.Second)
```

Comment:
What's happening here? We should be using tabs everywhere to be consistent with Golang